test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -277,6 +278,7 @@ static inline void multi_access_publisher(bool intra_process)
 
   const size_t num_messages = 5 * std::min<size_t>(executor.get_number_of_threads(), 16);
   auto pub = node->create_publisher<test_rclcpp::msg::UInt32>(node_topic_name, num_messages);
+  printf("Publisher initialized");
```

Comment:
Mind removing this print statement? It seems unrelated.

Reply:
Done